src/components/themed/TransactionListRow.tsx

```diff
@@ -83,6 +94,16 @@ export function TransactionListRow(props: Props) {
   const cryptoAmount = div(abs(transaction.nativeAmount ?? '0'), displayDenomination.multiplier, DECIMAL_PRECISION)
   const cryptoAmountFormat = formatNumber(decimalOrZero(truncateDecimals(cryptoAmount, maxConversionDecimals), maxConversionDecimals))
 
+  // Fiat Amount
+  if (amountFiat === 0) {
```

Comment:
We need a third possible state since 0 is a valid amount for some currencies. The initial value might need to be (number | undefined) so that the historical rate query can set it to 0.

Reply:
I thought about that, problem is we have a bunch of 0 value transactions saved by the Core. We want to make sure to override those with a new value.